### PR TITLE
Lots of refactoring...

### PR DESCRIPTION
**Pull request by lintool**
Major changes:
- moved regression data to data/
- switched over the latest version of Cloud9
- general code cleanup

### CODE REVIEW

File: src/java/main/ivory/data/IntPostingsForwardIndex.java [hunk 1]
```diff
@@ -67,38 +65,41 @@ public class IntPostingsForwardIndex {
...
-		FSDataInputStream posInput = mFs.open(new Path(fwindexPath));
-
-		int l = posInput.readInt();
-		positions = new long[l];
-		for (int i = 0; i < l; i++) {
-			positions[i] = posInput.readLong();
-			//sLogger.info(positions[i]);
-		}
-	}
+//	public IntPostingsForwardIndex(String postsPath, String fwindexPath, FileSystem fs, String type) throws IOException {
```
Comment (metzlerd) [inline src/java/main/ivory/data/IntPostingsForwardIndex.java:68]: Entire constructor commented out -- should it just be removed?

File: src/java/main/ivory/data/IntPostingsForwardIndex.java [hunk 2]
```diff
@@ -67,38 +65,41 @@ public class IntPostingsForwardIndex {
...
-		FSDataInputStream posInput = fwindexFS.open(new Path(fwindexPath));
-
-		int l = posInput.readInt();
-		positions = new long[l];
-		for (int i = 0; i < l; i++) {
-			positions[i] = posInput.readLong();
-			//sLogger.info(positions[i]);
-		}
-	}
+//	public IntPostingsForwardIndex(String postsPath, FileSystem postsFS, String fwindexPath, FileSystem fwindexFS) throws IOException {
```
Comment (metzlerd) [inline src/java/main/ivory/data/IntPostingsForwardIndex.java:84]: Entire constructor commented out -- should it just be removed?

Comment (lintool) [inline src/java/main/ivory/data/IntPostingsForwardIndex.java:84]: How I typically deal with "unused code". First, I comment out. Then a wait a bit (e.g., for people to complain---there maybe downstream folks who are using exported jars). A bit later I physically remove. This is a bit harsher than adding a deprecation tag, but basically the same idea.

**Comment by metzlerd** [comment]
Everything looks reasonable to me.

**Comment by lintool** [comment]
Will merge to master. Closing issue.

**Pull request closed without merging**